docs/core/integration-quality-scale/rules/iqs045.md

```diff
@@ -0,0 +1,42 @@
+---
+title: "IQS045 - The documentation describes all integration configuration options"
+---
+
+IQS045 is as follows:
+
+The documentation describes all integration configuration options
+
+## Reasoning
+
+Integrations can provide an options flow to allow users to change integration configuration.
+This rule ensures that all configuration options are documented so that users can understand what each option does and how to use it.
+
+## Example implementation
+
+The following example is for an integration with multiple configuration options, using the `configuration_basic` tag.
+
+```markdown
+## Configuration options
+
+The integration provides the following configuration options:
+
+{% configuration_basic %}
+Country code:
+  description: You can specify the country code (NL or BE) of the country to display on the camera.
+Delta:
+  description: Time interval in seconds between camera image updates
```

Comment:
I think we should not include this option. I'm not sure we'd approve it today, and it's not good to give wrong impressions what are ok options.
```suggestion
```